src/main/java/module/youth/YouthPlayerOverviewTableModel.java

```diff
@@ -19,6 +22,10 @@
 
 public class YouthPlayerOverviewTableModel extends HOTableModel {
 
+    private final Color aboveAverageRatingColor = ThemeManager.getColor(HOColorName.YOUTH_ABOVE_AVERAGE_RATING);
+    private final Color highRatingColor = ThemeManager.getColor(HOColorName.YOUTH_HIGH_RATING);
+    private final Color highestRatingColor = ThemeManager.getColor(HOColorName.YOUTH_HIGHEST_RATING);
```

Comment:
Could these be made local variables in `initColumns`?  There is not much point in making these static constants in that particular class.

Reply:
Good point - changed this.